CMakeLists.txt

```diff
@@ -202,6 +202,7 @@ option(FLB_IN_OPENTELEMETRY            "Enable OpenTelemetry input plugin"
 option(FLB_IN_ELASTICSEARCH            "Enable Elasticsearch (Bulk API) input plugin" Yes)
 option(FLB_OUT_AZURE                   "Enable Azure output plugin"                   Yes)
 option(FLB_OUT_AZURE_BLOB              "Enable Azure output plugin"                   Yes)
+option(FLB_OUT_AZURE_LOGS_INGESTION    "Enable Azure Logs Ingestion output plugin"    Yes)
```

Comment:
Does this definitely work for all targets (e.g. Windows and Mac too)? 
We need to ensure the defaults for each platform work.

Reply:
it's fine, looks like it built - I used the CI to do it